<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="main_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>671</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>991</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#main_frame {
background-color: #081E45;}



</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_browse">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #99C1F1;
color:  #000000;
font: 500 11pt &quot;Fira Sans&quot;;
</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_predict">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #99C1F1;
color:  #000000;
font: 500 11pt &quot;Fira Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_H1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 400 30pt &quot;Fira Sans&quot;;
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Paddy Lab </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_img_view">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>450</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #081E39;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #99C1F1;
color:  #000000;
font: 500 11pt &quot;Fira Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_message">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>450</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #081E39;
color: #ffffff;
font: 300 11pt &quot;Fira Sans&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_main_icon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>19</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_H2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 13pt &quot;Fira Sans&quot;;
color: #C0BFBC;</string>
    </property>
    <property name="text">
     <string> A Deep Learning tool for Paddy disease identification</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_background_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #081E39;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lbl_background_btn</zorder>
   <zorder>btn_browse</zorder>
   <zorder>btn_predict</zorder>
   <zorder>lbl_H1</zorder>
   <zorder>lbl_img_view</zorder>
   <zorder>btn_clear</zorder>
   <zorder>lbl_message</zorder>
   <zorder>lbl_main_icon</zorder>
   <zorder>lbl_H2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
